### PR TITLE
WebGLUniforms: Add unsigned vector array support for uniform.

### PR DESCRIPTION
**Pull request by SuperSodaSea**
**Description**

Writing unsigned vector uniforms in shaders:
```GLSL
uniform uvec2 foo;
```

And using them can result in this exception:

```
TypeError: u.setValue is not a function
```

I checked out the code, and found that the `WebGLUniforms` didn't take care of `UNSIGNED_INT_VEC2/3/4`. This pull request add code to support them.

### CODE REVIEW

File: src/renderers/webgl/WebGLUniforms.js [hunk 1]
```diff
@@ -414,7 +414,7 @@ function setValueT6( gl, v, textures ) {
 
 }
 
-// Integer / Boolean vectors or arrays thereof (always flat arrays)
+// Integer / Boolean
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLUniforms.js:417]: Um, why have to deleted `vectors or arrays thereof (always flat arrays)`?

Comment (SuperSodaSea) [reply src/renderers/webgl/WebGLUniforms.js:417]: This comment seems to be directly copied from #L607, where the code about array occurs, and the code here is only about single integer / bool (vectors at #L431), so I changed the incorrect comment.

Comment (Mugen87) [reply src/renderers/webgl/WebGLUniforms.js:417]: The comment refers to all call `setValueV*i()` calls. Starting from `setValueV2i()`, `v` is an array and `arraysEqual()` is used.

Comment (SuperSodaSea) [reply src/renderers/webgl/WebGLUniforms.js:417]: I suppose the "arrays" in the comment refers to `setValueV*iArray`, aka  `uniform uvec2 foo[42];`. The "always flat arrays" is a contrast to "flat or from THREE classes": `flatten()` in `setValueV*fArray`. I think it is a mistake.

Comment (SuperSodaSea) [reply src/renderers/webgl/WebGLUniforms.js:417]: Using git blame I found that the mistake is from [this](https://github.com/mrdoob/three.js/commit/22c2356853bcd06a0912dd470056a34a4012b86e#diff-ae819466109851758b72993ae3f4d77a3a1835d343ddf1f9dfa0c7a791b65a6cR504) commit.
The `setValue*iv` is splited into `setValueV*i` and `setValueV*ia`, then the comment is copied twice but forgett to change.

Comment (Mugen87) [reply src/renderers/webgl/WebGLUniforms.js:417]: Please use the same pattern like `setValueV*f()` is documented.

Comment (SuperSodaSea) [reply src/renderers/webgl/WebGLUniforms.js]: In the [next](https://github.com/mrdoob/three.js/pull/21316/commits/98b3de4a23a71c128c91ddf0084417535e772c69) commit, the comments are rewritten, and the order of the `setValue*` funcitons is also changed to fit their order in `getSingularSetter` and `getPureArraySetter`.

**Review by Mugen87 — APPROVED** [review]

**Comment by WestLangley** [comment]
@Mugen87 FYI, I added the missing milestone.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**